kong/plugins/request-transformer/schema.lua

```diff
@@ -10,8 +10,18 @@ local function check_for_value(value)
   return true
 end
 
+local function check_method(value)
+  if not value then return true end
+  local method = value:upper()
+  local ngx_method = ngx["HTTP_"..method]
+  if not ngx_method then
+    return false, method.." is not supported"
```

Comment:
Think of this value as being stored into a `is_valid` property, that's why it's `true` or `false`.

```lua
local function check_method(value)
  if not value then return true end
  local method = value:upper()
  local ngx_method = ngx["HTTP_"..method]
  if not ngx_method then
    return false, method.." is not supported"
  end
  return true
end

local is_valid, err = check_method("hello")
```

Makes sense that the return value is `boolean`, with an optional error message into `err`.